pulsar-broker/src/main/java/org/apache/pulsar/broker/resourcegroup/ResourceGroupService.java

```diff
@@ -651,7 +654,10 @@ private void checkRGCreateParams(String rgName, org.apache.pulsar.common.policie
     }
 
     private static final Logger log = LoggerFactory.getLogger(ResourceGroupService.class);
+
+    @Getter
     private final PulsarService pulsar;
+
     protected final ResourceQuotaCalculator quotaCalculator;
     private ResourceUsageTransportManager resourceUsageTransportManagerMgr;
     private final ResourceGroupConfigListener rgConfigListener;
```

Comment:
Qs/nit [unrelated to this specific PR]: the rgConfigListener  is set up in the ResourceGroupService() ctors, but I didn't see its usage.